system-int-test/test/cmr/system_int_test/search/collection_directory_pages_test.clj

```diff
@@ -0,0 +1,242 @@
+(ns cmr.system-int-test.search.collection-directory-pages-test
+  "This tests a running CMR site's directory links at all levels: top-most,
+  eosdis, and provider."
+  (:require [clj-http.client :as client]
+            [clojure.test :refer :all]
+            [clojure.string :as string]
```

Comment:
After a quick chat, it seems like we're moving towards more standard require-as aliases ...